x-pack/plugins/observability/public/pages/slos/index.tsx

```diff
@@ -67,7 +67,12 @@ export function SlosPage() {
           defaultMessage: 'SLOs',
         }),
         rightSideItems: [
-          <EuiButton color="primary" fill onClick={handleClickCreateSlo}>
+          <EuiButton
+            color="primary"
+            fill
+            onClick={handleClickCreateSlo}
+            data-test-subj="slosPage-createNewSloButton"
```

Comment:
nice! this will track via EBT whenever a user creates a new SLO 👍 

And I assume edit/clone/delete already have their own `data-test-subj`, correct?

Reply:
Correct, the other actions had already `data-test-subj`